hazelcast/src/main/java/com/hazelcast/spi/OperationFactory.java

```diff
@@ -26,9 +26,8 @@
  * {@link OperationService} uses this factory to create {@link Operation}s for each partition by calling
  * {@link #createOperation()}
  * </p>
- *
  */
-public interface OperationFactory extends DataSerializable {
```

Comment:
I would say that it's better to break & fix it now. Otherwise we're stucked with the package and naming forever.
It would be great to have a policy banning the usage of DS in HZ classes guarded by a checkstyle/unittest rule run in each build. Otherwise it will be constant struggle.